pkg/util/log/audit/audit.go

```diff
@@ -65,125 +72,138 @@ var (
 
 // NewEntry returns a log entry that embeds the provided logger. It has a hook
 // that knows how to hydrate an IFxAudit log payload before logging it.
-func NewEntry(env env.Core, logger *logrus.Logger) *logrus.Entry {
+func NewEntry(logger *logrus.Logger) *logrus.Entry {
```

Comment:
yeah, yuck! that's pretty gross! how about we pass in a value, instead of a pointer? moving this into `util.GetAuditLogger()` still ends up mutating the base logger anyway. i prefer to still have a constructor as from a OOD perspective, it doesn't make sense that the caller must depend on `util` to use this package.